### DISCUSSION TITLE
Fix arrangement of file names in columns

### DISCUSSION THREAD

**Discussion by abhijitnathwani** [initialcommit-com/git-sim#68] [Q&A] [answered]
Hi @initialcommit-io 
I observed one thing when I run `git-sim add <file>`, andyou have files in working mods and untracked files, the arrow from column 1 to column 3 is drawn over the file names in column 2. This is not definitive, but running the same command multiple times gives the problematic visualization.

Normal simulation:
![git-sim-add_03-05-23_15-34-11](https://user-images.githubusercontent.com/5513984/222954273-5bde143e-70a3-4930-9f80-4ad8c9fc9e29.jpg)

Errorneus simulation:
![git-sim-add_03-05-23_15-34-26](https://user-images.githubusercontent.com/5513984/222954298-3297cbb5-1244-4227-bba7-be40a890aabd.jpg)

I guess something to do with the arrangement of file names in column 1. Can you have a look?

**Comment by initialcommit-io** [comment, marked as answer]
@abhijitnathwani Good find - yes I was actually aware of this one but haven't gotten around to fixing it yet.  I think this belongs better as an Issue than a discussion.  Can you create an issue for it?

**Comment by initialcommit-io** [comment]
Closing since we have issue #71 for this now.
